src/web/wfs/src/test/java/org/geoserver/wfs/web/WFSAdminPageTest.java

```diff
@@ -53,6 +53,13 @@ public void testChangesToValues() throws Exception {
         ft.submit("submit");
         wfs = getGeoServerApplication().getGeoServer().getService(WFSInfo.class);
         assertEquals("allowGlobalQueries = false", false, wfs.getAllowGlobalQueries());
+        // test includeWFSRequestDumpFile
+        tester.startPage(WFSAdminPage.class);
+        ft = tester.newFormTester("form");
+        ft.setValue("includeWFSRequestDumpFile", false);
+        ft.submit("submit");
+        wfs = getGeoServerApplication().getGeoServer().getService(WFSInfo.class);
+        assertEquals("includeWFSRequestDumpFile= false", false, wfs.getIncludeWFSRequestDumpFile());
```

Comment:
Should have been 

```suggestion
        assertFalse("includeWFSRequestDumpFile= false", wfs.getIncludeWFSRequestDumpFile());
```

(a PMD check will likely fail this line, after you get formatting going)